src/main/java/com/github/dockerjava/api/command/ExecCreateCmd.java

```diff
@@ -23,6 +24,9 @@
     @CheckForNull
     String getUser();
 
+    @CheckForNull
+    List<String> getEnv();
```

Comment:
It has to be a Map; I used a List because of your suggestion.
All other ENV implementations in the project use String[]...
Maybe, you can accept this pull request, and plan together a refactor to change all ENV to Map's, to avoid repeating ENV VARS fields and to make it clearer and usable.